### DISCUSSION TITLE
About the properties of Get-MgUserCalendarView

### DISCUSSION THREAD

**Discussion by kop-7777** [microsoftgraph/msgraph-sdk-dotnet#2682] [Q&A]
Since EWS is being discontinued, I am trying to change it to MsGraph.
If I change the command line from EWS to Get-MgUserCalendarView, the properties will change to the following ones, so is this correct?
Please let me know.
[Properties]
ISAllDayEvent → IsAllDay
IsReminderSet → IsReminderOn
Resources → Attendees

**Comment by andrueastman** [comment]
Thanks for raising this @kop-7777 

I believe the command let you are calling for graph will return an object of the event type. https://learn.microsoft.com/en-us/graph/api/resources/event?view=graph-rest-1.0

With regards to direct mapping of the properties from EWS, any chance you can post the question at the link below to get feedback from API owners? As these repos are mainly intended for Graph SDKs we may not be best placed to answer API related questions.

https://aka.ms/askGraph

**Reply by kop-7777** [reply]
Thanks for letting me know @andrueastman 

I'll try asking the question on the site you directed me to.
